client/me/help/help-contact/index.jsx

```diff
@@ -518,7 +540,7 @@ const HelpContact = React.createClass( {
 					showSubjectField: false,
 					showHowCanWeHelpField: false,
 					showHowYouFeelField: false,
-					showSiteField: hasMoreThanOneSite,
+					showSiteField: false,
```

Comment:
@dllh Due to rebases and code merges this morning this PR isn't reading clean. This change is already merged into master and this branch needs a rebase, which I'll do in just a moment.